docs/reference/ml/df-analytics/apis/evaluate-dfanalytics.asciidoc

```diff
@@ -246,4 +249,85 @@ only. It means that a testing error will be calculated.
 <2> The field that contains the ground truth value for the actual student 
 performance. This is required in order to evaluate results.
 <3> The field that contains the predicted value for student performance 
-calculated by the {reganalysis}.
\ No newline at end of file
+calculated by the {reganalysis}.
+
+
+===== {classification-cap}
+
+
+[source,console]
+--------------------------------------------------
+POST _ml/data_frame/_evaluate
+{ 
+   "index": "animal_classification",
+   "evaluation": {
+      "classification": { <1>
+         "actual_field": "animal_class", <2>
+         "predicted_field": "ml.animal_class_prediction.keyword", <3>
+         "metrics": {  
+           "multiclass_confusion_matrix" : {}
+         }
+      }
+   }
+}
+--------------------------------------------------
+// TEST[skip:TBD]
+
+<1> The evaluation type.
+<2> The field that contains the ground truth value for the actual animal 
+classification. This is required in order to evaluate results.
+<3> The field that contains the predicted value for animal classification by 
+the {classanalysis}. Note that you need to add the `.keyword` suffix to the 
+name.
+
+
+The API returns the following result:
+
+[source,console-result]
+--------------------------------------------------
+{
+   "classification" : {
+      "multiclass_confusion_matrix" : {
+         "confusion_matrix" : [
+         {
+            "actual_class" : "cat", <1>
+            "actual_class_doc_count" : 12, <2>
+            "other_predicted_class_doc_count" : 0, <3>
```

Comment:
I think `other_predicted_class_doc_count` should come after `predicted_classes` stanza as it captures all the other docs that did not make it to `predicted_classes`.